src/index.html

```diff
@@ -9,7 +9,7 @@
   <link rel="stylesheet" href="assets/css/bootstrap-base.css">
   <!-- jQuery library -->
   <script src="//ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-  <script src="assets/chart.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.4.0/Chart.bundle.min.js"></script>
```

Comment:
it is good if we could serve this from our local system,please download the chart.bundle.min.js , save it locally and serve that.